### ISSUE TITLE
outlier labels of boxplots overlap

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-desktop#2535]
[exampleBox.zip](https://github.com/jasp-stats/jasp-desktop/files/2011952/exampleBox.zip)

Version 0.8.6 (flatpak)
Built on: Mar 22 2018 14:47:07 (Netherlands)
OS name and version: openSUSE Tumbleweed, Linux kernel 4.16.8-1 64bit, KDE
* Analysis: outlier labels of boxplots overlap and make them hard to read
* Steps to reproduce: produce a boxplot that shows nearby outliers like with the example data "Bugs" variable "Lo D Hi F"; mark " label outliers" within the "plots" section

enhancement requests
 
* Enhancement: make the labels non-overlapable somehow
* Purpose: better readability

**Comment by AlexanderLyNL** [comment]
@koenderks Can you check whether this is still the case. For the new plots?

**Comment by EJWagenmakers** [comment]
I think this will be difficult to fix in general (what do you think, @vankesteren?): the outliers could be anywhere, and there could be many of them. For instance, suppose you have a cluster of 20 outliers that are very close together. Tricky. Placing the labels so that they don't overlap will remove them from the points that they are associated with. 
E.J.

**Comment by tomtomme** [comment]
True. There would need to be a limit. Maybe only show labels for up to 4 outliers. Or only for discrete data. Or only for samples smaller than 100. Something like this. No general solution I know. But better than the current situation where all those numbers overlap

**Comment by vankesteren** [comment]
With the new plots we may be able to use something like [ggrepel](https://cran.r-project.org/web/packages/ggrepel/vignettes/ggrepel.html) to solve this issue in a quite general way?

**Comment by tomtomme** [comment]
@AlexanderLyNL 
I tried building jasp devel to see if this is still the case. However, I did not find a openSUSE build guide (only ubuntu / fedora) for jasp. The  "make" error messages I see indicate that I need to hunt missing devel packages. How can I find out which packages are still missing?
Qtcreator also is missing a project configuration and even the general examples are missing. So I am at a bit of a loss here but could try more, if you have suggestions.
